### ISSUE TITLE
[DocDB] xcluster: Automatically propagate DDL changes across clusters

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#11537]
### Description

We should eventually have a design for how to automatically apply DDL changes across clusters.
cc @nspiegelberg @rahuldesirazu @hulien22

**Comment by mrajcevic01** [comment]
@bmatican @nspiegelberg can we include the specific GH issues for the different phases of this project to the description?

**Comment by hulien22** [comment]
@abuzarkamal yes this feature is being actively worked on - this is just the parent issue, smaller issues are being worked on and tracked in the [xCluster DDL Replication project](https://github.com/yugabyte/yugabyte-db/projects/69)

**Comment by abuzarkamal** [comment]
Thanks for the update, Great progress

**Comment by abuzarkamal** [comment]
Thanks for the update, Great progress
